proxy-manager/build.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 build_from:
-  aarch64: ghcr.io/hassio-addons/base-nodejs:0.2.2
-  amd64: ghcr.io/hassio-addons/base-nodejs:0.2.2
-  armv7: ghcr.io/hassio-addons/base-nodejs:0.2.2
+  aarch64: ghcr.io/hassio-addons/base-nodejs:0.2.5
+  amd64: ghcr.io/hassio-addons/base-nodejs:0.2.5
+  armv7: ghcr.io/hassio-addons/base-nodejs:0.2.5
```

Comment:
I've restored the original base image in the main branch. Please rebase to get rid of this change.